torch/distributed/tensor/parallel/style.py

```diff
@@ -320,6 +277,28 @@ def make_output_tensor(
     ).to_local()  # type: ignore[call-arg]
 
 
+@_prepare_output_validate  # type: ignore[arg-type] # pyre-ignore[56]
+def make_sharded_output_tensor(
+    output: DTensor, _device_mesh: Optional[DeviceMesh] = None
+) -> torch.Tensor:
+    """
+    Convert sharded Output DTensor to Tensor.
+
+    Args:
+        output (:class:`DTensor`):
+            Output of module to be converted.
+        _device_mesh (:class:`DeviceMesh`, optional):
```

Comment:
Oh I see, maybe we don't really need to document this argument? Added a comment to explain this should be enough